LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -2128,6 +2128,56 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                             break;
                         }
+                        case Model.TUF_GAMING_B550M_PLUS_WIFI: //NCT6798D
+                        {
+                            v.Add(new Voltage("Vcore", 0));
+                            v.Add(new Voltage("Voltage #2", 1, true));
+                            v.Add(new Voltage("AVCC", 2, 34, 34));
+                            v.Add(new Voltage("+3.3V", 3, 34, 34));
+                            v.Add(new Voltage("Voltage #5", 4, true));
+                            v.Add(new Voltage("Voltage #6", 5, true));
+                            v.Add(new Voltage("Voltage #7", 6, true));
+                            v.Add(new Voltage("3VSB", 7, 34, 34));
+                            v.Add(new Voltage("VBat", 8, 34, 34));
+                            v.Add(new Voltage("VTT", 9));
+                            v.Add(new Voltage("Voltage #11", 10, true));
+                            v.Add(new Voltage("Voltage #12", 11, true));
+                            v.Add(new Voltage("Voltage #13", 12, true));
+                            v.Add(new Voltage("Voltage #14", 13, true));
+                            v.Add(new Voltage("Voltage #15", 14, true));
+                            t.Add(new Temperature("System", 0));
+                            t.Add(new Temperature("CPU", 1));
+                            t.Add(new Temperature("AUX 0", 2));
+                            t.Add(new Temperature("AUX 1", 3));
+                            t.Add(new Temperature("AUX 2", 4));
+                            t.Add(new Temperature("AUX 3", 5));
+                            t.Add(new Temperature("AUX 4", 6));
+                            t.Add(new Temperature("SMBus 0", 7));
+                            t.Add(new Temperature("SMBus 1", 8));
+                            t.Add(new Temperature("PECI 0", 9));
+                            t.Add(new Temperature("PECI 1", 10));
+                            t.Add(new Temperature("PCH Chip CPU Max", 11));
+                            t.Add(new Temperature("PCH Chip", 12));
+                            t.Add(new Temperature("PCH CPU", 13));
+                            t.Add(new Temperature("PCH MCH", 14));
+                            t.Add(new Temperature("Agent 0 Dimm 0", 15));
+                            t.Add(new Temperature("Agent 0 Dimm 1", 16));
+                            t.Add(new Temperature("Agent 1 Dimm 0", 17));
+                            t.Add(new Temperature("Agent 1 Dimm 1", 18));
+                            t.Add(new Temperature("Byte 0", 19));
+                            t.Add(new Temperature("Byte 1", 20));
```

Comment:
Is this even useful to add?

Reply:
It's in the list of possible temperature source, but the docs again do not indicate what it purpose might be. We may rename it to Device0 and Device1, which would also match the docs but still doesn't tell us anything useful.

Reply:
OK, let's keep it then 👍

Reply:
Again the docs aren't helpful at all here. They are in the list of possible temperature sources. They could also be called Device 0 and Device 1, but no infos about what devices that may be. And again they're not used on my board.

Reply:
I think Device 0 and Device 1 is better, so I'll rename them, too.